example/src/tests.ts

```diff
@@ -0,0 +1,57 @@
+import { Wallet } from "ethers";
+import { Client } from "xmtp-react-native-sdk";
+import * as XMTP from "../../src/index";
+
+function sleep(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export type Test = {
+  name: string;
+  run: () => Promise<boolean>;
+};
+
+export const tests: Test[] = [];
+
+function test(name: string, perform: () => Promise<boolean>) {
+  tests.push({ name, run: perform });
+}
+
+// test("can fail", async () => {
```

Comment:
nit: leftover commented code

Reply:
Leaving this around for now just so I can uncomment it when I set up CI and make sure that's all working. Good eye though!